pages/api/checkout/session.ts

```diff
@@ -0,0 +1,41 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import Stripe from "stripe";
+
+// set up stripe instance
+// TODO: set this as exportable variable in utils directory
+const stripe = new Stripe(process.env.STRIPE_SECRET_KEY, {
+    apiVersion: "2020-08-27",
+});
+
+/**
+ * sessionHandler creates a new stripe session and returns the
+ * session ID
+ * @param req API request object
+ * @param res API response object
+ */
+export default async function sessionHandler(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    if (req.method == "POST") {
+        const { quantity } = req.body;
+        // TODO: obtain price id as part of body
+        const session = await stripe.checkout.sessions.create({
+            payment_method_types: ["card"],
+            line_items: [
+                {
+                    price: process.env.PRICE_ID,
```

Comment:
What's PRICE_ID?

Reply:
It's the ID for the product in Stripe. You can create different products in Stripe with different prices and that price ID is use for redirecting the user to the correct checkout screen.

Reply:
Could you explain why we need to obtain the price id as part of body (in the TODO comment)?

Reply:
Yeah - I was thinking each paid class could be a separate product with a unique `PRICE_ID`. That way, we can just pass that PRICE_ID into the CheckoutButton component for the whichever class the user wants to sign up for.